src/modm/driver/touch/touch2046.hpp

```diff
@@ -0,0 +1,134 @@
+// coding: utf-8
+/*
+ * Copyright (c) 2021, Raphael Lehmann
+ *
+ * This file is part of the modm project.
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+// ----------------------------------------------------------------------------
+
+#ifndef MODM_TOUCH2046_HPP
+#define MODM_TOUCH2046_HPP
+
+#include <modm/architecture/interface/spi_device.hpp>
+#include <modm/architecture/interface/gpio.hpp>
+#include <modm/processing/resumable.hpp>
+
+namespace modm
+{
+
+/// \ingroup modm_driver_touch2046
+struct touch2046 {
+	enum class Orientation {
+		Normal,
+		//...
+	};
+
+	/**
+	 * \brief	Calibration values are used to calculate touch point
```

Comment:
We don't need the `\brief`/`@brief` keywords, since we have autobrief enabled. Just FYI before you copy-paste that everywhere ;-P

Reply:
Nice to known.